.travis.yml

```diff
@@ -17,7 +15,7 @@ env:
     - ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
 install:
   - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
-script:
-  - source .ci_config/travis.sh
+script: 
+  - .ci_config/travis.sh
```

Comment:
I like this notation here better, than the one in `indigo_dev` (see https://github.com/ipa320/cob_command_tools/pull/179/files#diff-354f30a63fb0907d4ad57269548329e3R17)!
(except for the trailing whitespace :wink:)